site/index.md

```diff
@@ -0,0 +1,213 @@
+
+As SBOM usage evolves we will see new and different guidance on this topic. Check back in regularly as we will continue to update this site as the industry helps define how to create and use SBOMs.
+
+#### Source
+The source SBOM represents your application during development. This SBOM
+could contain development dependencies, development versions of open source
+packages, and software needed to build an application. It might contain files that only exist in certain branches of your source management system.
+
+A source SBOM gives you the ability to understand if any of your
+dependencies are outdated (including development dependencies, these often
+get ignored). You can also use source SBOMs as a sort of time machine to
+look back to when a certain file or dependency was first added to the
+project.
+
+#### Build
+The build SBOM is the SBOM that you generate during a build. It could be a
+partial build, or a test build, or even the final build. By generating an
+SBOM during the build stage you can look back into what exactly was built
+and shipped for any product or component.
+
+This SBOM will differ from the source SBOM because during a build things
+can happen that are outside of a development environment. Maybe a container
+is created. A binary could be copied into the build. Builds can be signed
+during this stage, the signatures and checksums can also be captured by the
+SBOM scanner.
+
+#### Runtime
+
+A runtime SBOM is the thing that will be deployed into an environment. It
+will contain final builds, supporting libraries (like an operating system
+or container image). The configuration files have been updated, defaults
+have probably changed. It's even possible someone modified some of the
+contents that we were tracking in the build SBOM.
+
+The runtime SBOM can help us understand when our application contains
+security vulnerabilities. It can give us insight into what we've changed in
+our running deployment vs what we received from our vendor. It can also
+give us insight into understanding how old our application and dependencies
+are.
+
+## How can I make an SBOM?
+
+The single easiest way to create an SBOM is to use a tool called
+[Syft](https://github.com/anchore/syft/)
+
+There are a variety of ways to install Syft, please see these
+[instructions](https://github.com/anchore/syft/#installation) for the best
+way on your system.
+
+Once Syft is installed, it's very easy to run.
+
+We are going to base these examples on Syft itself. The first thing we will
+do is pull the Syft GitHub repository then scan the Syft source code.
+
+`git clone https://github.com/anchore/syft.git`
+
+### Scanning a directory
+
+You can scan many different types of artifacts with Syft. In this instance we will scan a directory with Syft. We should scan the repository we just
+checked out.
+
+First run
+```
+➜  ~ syft  dir:./syft
+```
+
+This command will give us a lot of output. This is our source SBOM. There
```

Comment:
nit: maybe specify that this command does not generate an sbom, but the table summary?